minio/minio
# Added `endpoint` and `versions` attributes to KMS details

## Description
Now it would list details of all KMS instances with additional attributes `endpoint` and `version`. In case of k8s based deployment the list would consist a single entry.

## Motivation and Context
In case there are multiple instances of KES deployed and they have different versions, this would make sure details are listed at instance level.

## How to test this PR?
Needs: https://github.com/minio/mc/pull/4591 and https://github.com/minio/madmin-go/pull/197

`mc admin info ALIAS --json` should show `.info.services.kmsStatus` as below

```
$ ./mc admin info myminio --json | jq '.info.services.kmsStatus'
[
  {
    "status": "online",
    "encrypt": "success",
    "decrypt": "success",
    "endpoint": "https://127.0.0.1:7373",
    "version": "2023-05-11T04-04-40Z"
  },
  {
    "status": "online",
    "encrypt": "success",
    "decrypt": "success",
    "endpoint": "https://127.0.0.1:7474",
    "version": "2023-05-11T04-04-40Z"
  }
]
```

## Types of changes

## Checklist:
